<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportAnnotationsWizard</class>
 <widget class="QWizard" name="ExportAnnotationsWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ModernStyle</enum>
  </property>
  <widget class="QWizardPage" name="pageCorpusItemsSelection">
   <property name="title">
    <string>Select Corpus Items to Export</string>
   </property>
   <property name="subTitle">
    <string>Select the corpus items you want to export. Selecting a Communication will export all the associated Annotations.</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="3">
     <layout class="QGridLayout" name="gridLayoutTop">
      <item row="1" column="2">
       <widget class="QPushButton" name="commandSelectFileFolder">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2">
       <layout class="QGridLayout" name="gridLayoutExport">
        <item row="0" column="0">
         <widget class="QComboBox" name="comboBoxExportFormat">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="editFileFolder"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelFileFolder">
        <property name="text">
         <string>Folder:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelExportFormat">
        <property name="text">
         <string>Export format:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelRepository">
      <property name="text">
       <string>Export annotations from Corpus Repository: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBoxRepository"/>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>353</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBoxAnnotations">
      <property name="title">
       <string>Select Annotations to Export</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTreeView" name="treeViewAnnotations"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBoxExportFormat</tabstop>
  <tabstop>editFileFolder</tabstop>
  <tabstop>commandSelectFileFolder</tabstop>
  <tabstop>comboBoxRepository</tabstop>
  <tabstop>treeViewAnnotations</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
